read_input.l

```diff
@@ -2596,6 +2877,16 @@ int read_input(char * conf_file){
   g_N_s = _default_g_N_s;
   g_dflgcr_flag = _default_g_dflgcr_flag;
 
+/* Setting the default parameters for QUDA multigrid inverter */
+  Quda_multigrid_input.nlevel=3;
+  Quda_multigrid_input.nvec  =24;
+  Quda_multigrid_input.nu_post =4;
+  Quda_multigrid_input.nu_pre  =4;
+  Quda_multigrid_input.blocksize[0][0]=0;
```

Comment:
So, do I understand it correctly that it will attempt to use the blocking algorithm that I came up with if no block sizes are passed?